vclib-openid/src/commonTest/kotlin/at/asitplus/wallet/lib/oidc/DummyOAuth2IssuerCredentialDataProvider.kt

```diff
@@ -50,33 +50,25 @@ class DummyOAuth2IssuerCredentialDataProvider(
                 optionalClaim(claimNames, "subject", userInfo.userInfo.subject),
                 userInfo.userInfo.birthDate?.let { optionalClaim(claimNames, "date-of-birth", it) },
             )
-            credentials += when (representation) {
-                ConstantIndex.CredentialRepresentation.SD_JWT -> listOf(
-                    CredentialToBeIssued.VcSd(
-                        claims = claims,
-                        expiration = expiration,
-                    )
+            when (representation) {
+                ConstantIndex.CredentialRepresentation.SD_JWT -> CredentialToBeIssued.VcSd(
+                    claims = claims,
+                    expiration = expiration,
                 )
 
-                ConstantIndex.CredentialRepresentation.PLAIN_JWT -> claims.map { claim ->
-                    CredentialToBeIssued.VcJwt(
-                        subject = AtomicAttribute2023(subjectId, claim.name, claim.value.toString()),
-                        expiration = expiration,
-                    )
-                }
+                ConstantIndex.CredentialRepresentation.PLAIN_JWT -> CredentialToBeIssued.VcJwt(
+                    subject = AtomicAttribute2023(subjectId, "given_name", givenName ?: "no value"),
+                    expiration = expiration,
+                )
 
-                ConstantIndex.CredentialRepresentation.ISO_MDOC -> listOf(
-                    CredentialToBeIssued.Iso(
-                        issuerSignedItems = claims.mapIndexed { index, claim ->
-                            issuerSignedItem(claim.name, claim.value, index.toUInt())
-                        },
-                        expiration = expiration,
-                    )
+                ConstantIndex.CredentialRepresentation.ISO_MDOC -> CredentialToBeIssued.Iso(
+                    issuerSignedItems = claims.mapIndexed { index, claim ->
+                        issuerSignedItem(claim.name, claim.value, index.toUInt())
+                    },
+                    expiration = expiration,
                 )
             }
-        }
-
-        if (credentialScheme == MobileDrivingLicenceScheme) {
+        } else if (credentialScheme == MobileDrivingLicenceScheme) {
```

Comment:
Please refactor to `when`-branching to conform to kotlin coding guidelines:
https://kotlinlang.org/docs/coding-conventions.html#if-versus-when